src/phone.ts

```diff
@@ -60,4 +60,17 @@ export class Phone {
       this.faker.definitions.phone_number.formats
     );
   }
+
+  /**
+   * Returns a random IMEI number.
```

Comment:
```suggestion
   * Generates IMEI number.
```

In faker everything is random :)